### DISCUSSION TITLE
There are no commands defined in the "ray" namespace.

### DISCUSSION THREAD

**Discussion by SirNovi** [spatie/laravel-ray#49] [Q&A]
Yesterday I installed ray in my Laravel 8 app and it all worked fine. 
Today it doesn't work anymore. I wanted to maybe have a look at the config but trying to publish it I got

`There are no commands defined in the "ray" namespace. `

How could this have gone awry?

**Comment by freekmurze** [comment]
It seems to be working for me, could you share a small repo in which you reproduce the problem?

**Comment by SirNovi** [comment]
Nevermind. `composer update` fixed it. 
Sorry for the bother.
